src/components/reusuables/sections/YohdevSection.js

```diff
@@ -0,0 +1,47 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import YohDevButton from "../../../components/reusuables/YohDevButton"
+import SectionHeader from "../typography/SectionHeader"
+
+const YohdevSection = (props) => {
+  const sectionHeading = props.sectionHeading == true ? true : false
+  const sectionHeadingAlt = props.sectionHeadingAlt === true ? "d-md-flex" : ""
+  const textContainerStyle = props.textContainerColor == "white" ? "white" : ""
+  const sectionContainerStyle =
+    props.sectionContainerStyle == "default"
+      ? "default"
+      : props.sectionContainerStyle == "secondary"
+      ? "secondary"
+      : props.sectionContainerStyle == "split"
+      ? "split"
+      : ""
+
+  return (
+    <section className={`yohdev-section-container ${sectionContainerStyle}`}>
+      <Container>
+        {sectionHeading && (
+          <div
+            className={`text-container mb-5 ${
+              (textContainerStyle, sectionHeadingAlt)
+            }`}
+          >
+            <SectionHeader heading={props.heading} />
+            <p className={`section-text`}>{props.text}</p>
+          </div>
+        )}
+
+        {props.children}
+
+        {props.button == "true" ? (
```

Comment:
This should be something like button && { }. "true" prop should be replaced with Boolean true.